test/case/infix_dhcp/client_hostname/topology.dot

```diff
@@ -0,0 +1,23 @@
+graph "1x2" {
+	layout="neato";
+	overlap="false";
+	esep="+100";
+
+        node [shape=record, fontname="DejaVu Sans Mono, Book"];
+	edge [color="cornflowerblue", penwidth="2", fontname="DejaVu Serif, Book"];
+
+	host [
+	    label="host | { <mgmt> mgmt | <data> data }",
+	    pos="0,20!",
+	    requires="controller",
+	];
+
+        client [
```

Comment:
Use one link instead of two, to be able to run test on units with only one port.